lib/App/Sqitch/Engine.pm

```diff
@@ -1572,10 +1572,6 @@ The name of the registry schema or database.
 
 The point in the plan from which to start deploying changes.
 
-=head3 C<no_prompt>
-
-Boolean indicating whether or not to prompt for reverts. False by default.
-
```

Comment:
Ugh, did we never document `prompt_accept`?

Reply:
It seems that way?